### ISSUE TITLE
 ninja: error: build.ninja:120: bad $-escape (literal $ must be written as $$)

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [dyninst/dyninst#1076]
**Intention**
make an ebuild for the gentoo oevrlay guru

**Describe the bug**
```
-- Configuring done
-- Generating done
CMake Error:
  Running

   '/usr/bin/ninja' '-C' '/var/tmp/portage/sys-cluster/dyninst-11.0.1/work/dyninst-11.0.1_build' '-t' 'recompact'

  failed with:

   ninja: error: build.ninja:120: bad $-escape (literal $ must be written as $$)
```

**To Reproduce**
`cmake -C /var/tmp/portage/sys-cluster/dyninst-11.0.1/work/dyninst-11.0.1_build/gentoo_common_config.cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/usr -DSTERILE_BUILD=YES -DENABLE_STATIC_LIBS=NO -DADD_VALGRIND_ANNOTATIONS=yes -DBUILD_DOCS=no -DUSE_OpenMP=yes -DCMAKE_BUILD_TYPE=Gentoo -DCMAKE_TOOLCHAIN_FILE=/var/tmp/portage/sys-cluster/dyninst-11.0.1/work/dyninst-11.0.1_build/gentoo_toolchain.cmake  /var/tmp/portage/sys-cluster/dyninst-11.0.1/work/dyninst-11.0.1`

It doesn't happen if I set `-DBUILD_RTLIB=NO`

**Expected behavior**
build successfully

**System (please complete the following information):**
gentoo

full log here https://gist.github.com/Alessandro-Barbieri/8ba9ad624e826ea7cef3048c6e6b7030

**Comment by Alessandro-Barbieri** [comment]
In `build.ninja:120` there is this line
`COMMAND = cd /var/tmp/portage/sys-cluster/dyninst-11.0.1/work/dyninst-11.0.1_build/dyninstAPI_RT && $(MAKE)`

**Comment by hainest** [comment]
I'm not familiar with using Ninja. Does CMake automatically set `MAKE` to something other than GNU Make when using Ninja as a generator in this context? I don't see that it could.

**Comment by jrmadsen** [comment]
This will be closed by #1101 

> Does CMake automatically set MAKE to something other than GNU Make when using Ninja as a generator in this context?

I'm pretty sure `$(MAKE)` used in that snippet was a Makefile variable (`MAKE := make`) in the Makefiles that CMake generated and thus errored bc it was Makefile syntax in a ninja script
